pkgs/development/libraries/aspell/aspell-with-dicts.nix

```diff
@@ -0,0 +1,31 @@
+# Create a derivation that contains aspell and selected dictionaries.
+# Composition is done using `pkgs.buildEnv`.
+
+{ aspell
+, buildEnv
+, makeWrapper
+, aspellDicts
+}:
+
+f:
+
+let
+  dicts = f aspellDicts;
+in buildEnv {
+  name = "aspell-env";
+  paths = dicts ++ [ aspell ];
+  buildInputs = [ makeWrapper ];
+  postBuild = ''
+#     wrapProgram $out/bin/aspell --set ASPELL_CONF "data-dir $out/lib/aspell"
+    # We cannot simply use wrapProgram because we're not allowed to modify items in $out/bin
+    unlink $out/bin
```

Comment:
ah, of course, removing individual items with `rm` doesn't work because the directory containing them is a symbolic link itself. I actually hadn't tried using `rm` on symbolic link to `/bin`.